### PR TITLE
OCC for maintenance

### LINKED ISSUE #395: Is OCC allowed here?
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 913-914
 
**Comment (Include rationale for comment)**: "The issuer SHALL perform a biometric verification…"
 
**Suggested Change**: "  E.g., card nearing expiration and re-issuance occurs.

What modalities?"
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### LINKED ISSUE #401: Is OCC allowed here?
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 1001-1003
 
**Comment (Include rationale for comment)**: "...positive biometric verification decision when compared to biometric data records stored either on the PIV Card or in the PIV enrollment record."
 
**Suggested Change**: "Is OCC allowed here?

What modalities/authentication modes are allowed here?"
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### LINKED ISSUE #584: Add "through an on-card one-to-one comparison" after "PIV Card".
### All Fields Are Required
**Organization Name (N/A, if individual)**: Secure Technology Alliance

**Organization Type (see below for codes)**: 2 - Industry
 
**Reference (Include section/paragraph or pdf line number)**: 2.9.3 para. 4 Line 1028
 
**Comment (Include rationale for comment)**: Same comment as above. This paragraph is about resetting a PIV card. If the PIV card needs to be reset, that means its PIN has been blocked due to too many consecutive failed verification attempts and therefore none of the biometric data on the card can be read.
 
**Suggested Change**: Add "through an on-card one-to-one comparison" after "PIV Card" in the sentence "… elicit a positive biometric verification decision when compared to biometric data records stored either on the PIV Card or in the PIV enrollment record." 
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

### PR DESCRIPTION
**Pull request by hferraio**
closes #395 
closes #401 
closes #584

### CODE REVIEW

File: _FIPS201/requirements.md [hunk 1]
```diff
@@ -406,13 +406,15 @@ If the expiration date of the new PIV Card is later than the expiration date of
...
 
-The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card or PIV enrollment record. Minimum accuracy requirements for
+The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card[^OCCorBIO] or PIV enrollment record. Minimum accuracy requirements for
 the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On a positive biometric verification decision, the new PIV Card **SHALL** be
 released to the applicant. If the biometric verification decision is negative, or if no biometric data records are available, the cardholder
 **SHALL** provide two identity source documents (as specified in [Section 2.7](requirements.md#s-2-7)), and an attending operator **SHALL**
 inspect these and compare the cardholder with the electronic facial image retrieved from the enrollment data record
 and the photograph printed on the new PIV Card.
 
+[^OCCorBIO]: BIO-A or OCC-AUTH authentication mechanism is performed when using the PIV Card for biometric  verification.
```
Comment (jimfenton) [inline _FIPS201/requirements.md:416]: But not BIO then, right? That makes this a normative requirement, and should be therefore inline in text.

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -497,17 +499,20 @@ remotely via a general computing platform or a supervised remote identity proofi
...
+    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored either on the PIV Card[^OCCorBIOv2] or in the PIV enrollment record. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
     to be reset and another primary identity source document (as specified in [Section 2.7](requirements.md#s-2-7)). An attending
     operator **SHALL** inspect these and compare the cardholder with the electronic facial image retrieved from the
     enrollment data record and the photograph printed on the card.
+    
+    [^OCCorBIOv2]: BIO-A or OCC-AUTH authentication mechanism is performed when using the PIV Card for biometric verification.
```
Comment (jimfenton) [inline _FIPS201/requirements.md:507]: See above

File: _FIPS201/requirements.md [hunk 3]
```diff
@@ -525,6 +530,8 @@ Supervised remote identity proofing station
     inspect these items and compare the video feed of the cardholder with the
     electronic facial image retrieved from the enrollment data record and the
     photograph printed on the PIV Card. 
+    
+     [^OCCorBIOv3]: BIO-A or OCC-AUTH authentication mechanism is performed when using the PIV Card for biometric verification.
```
Comment (jimfenton) [inline _FIPS201/requirements.md:534]: See above

**Review by jimfenton — CHANGES_REQUESTED** [review]
It looks like this puts a normative requirement in footnotes, which we're trying to avoid.

File: _FIPS201/requirements.md [hunk 4]
```diff
@@ -406,13 +406,14 @@ If the expiration date of the new PIV Card is later than the expiration date of
 about the cardholder is being changed, the card issuer **SHALL** ensure that an adjudicative entity has authorized
 the issuance of the new PIV Card. The issuer **SHALL** ensure that the adjudicative entity has verified that there is a PIV eligibility determination in an authoritative record, such as the agency's IDMS or the Central Verification System (or successor).
 
-The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card or PIV enrollment record. Minimum accuracy requirements for
+The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records of the PIV enrollment record or to the biometric data records of the  PIV card and performing BIO-A  or OCC-AUTH authentication mechanism. Minimum accuracy requirements for
```
Comment (gfiumara) [inline _FIPS201/requirements.md:409]:
```suggestion
The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records of the PIV enrollment record or to the biometric data records of the  PIV Card and performing BIO-A  or OCC-AUTH authentication mechanism. Minimum accuracy requirements for
```

File: _FIPS201/requirements.md [hunk 5]
```diff
@@ -516,7 +517,7 @@ Supervised remote identity proofing station
     procedure. The operator **SHALL** initiate a biometric verification to ensure that
     the cardholder's biometric characteristics captured at the station elicit a
     positive biometric verification decision when compared to biometric data
-    records stored either on the PIV Card or in the PIV enrollment record. In
+    records stored in the PIV enrollment record or when compared to the biometric data records on the PIV card and using BIO-A or OCC-AUTH authentication mechanism. In
```
Comment (gfiumara) [inline _FIPS201/requirements.md:520]:
```suggestion
    records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card and using BIO-A or OCC-AUTH authentication mechanism. In
```

File: _FIPS201/requirements.md [hunk 6]
```diff
@@ -497,17 +498,17 @@ remotely via a general computing platform or a supervised remote identity proofi
...
-    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored either on the PIV Card or in the PIV enrollment record. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
+    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using BIO-A or OCC-AUTH authentication mechanism. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
     to be reset and another primary identity source document (as specified in [Section 2.7](requirements.md#s-2-7)). An attending
     operator **SHALL** inspect these and compare the cardholder with the electronic facial image retrieved from the
     enrollment data record and the photograph printed on the card.
-
+
```
Comment (jricher) [inline _FIPS201/requirements.md:505]:
```suggestion

```

File: _FIPS201/requirements.md [hunk 7]
```diff
@@ -497,17 +498,16 @@ remotely via a general computing platform or a supervised remote identity proofi
...
-    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored either on the PIV Card or in the PIV enrollment record. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
+    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using BIO-A or OCC-AUTH authentication mechanism. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
     to be reset and another primary identity source document (as specified in [Section 2.7](requirements.md#s-2-7)). An attending
     operator **SHALL** inspect these and compare the cardholder with the electronic facial image retrieved from the
     enrollment data record and the photograph printed on the card.
-
 Issuer-operated kiosk
```
Comment (jricher) [inline _FIPS201/requirements.md:505]:
```suggestion

```

Comment (jricher) [inline _FIPS201/requirements.md:505]:
```suggestion

Issuer-operated kiosk
```

File: _FIPS201/requirements.md [hunk 8]
```diff
@@ -406,13 +406,14 @@ If the expiration date of the new PIV Card is later than the expiration date of
 about the cardholder is being changed, the card issuer **SHALL** ensure that an adjudicative entity has authorized
 the issuance of the new PIV Card. The issuer **SHALL** ensure that the adjudicative entity has verified that there is a PIV eligibility determination in an authoritative record, such as the agency's IDMS or the Central Verification System (or successor).
 
-The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card or PIV enrollment record. Minimum accuracy requirements for
+The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records of the PIV enrollment record or to the biometric data records of the  PIV Card and performing BIO-A  or OCC-AUTH authentication mechanism. Minimum accuracy requirements for
```
Comment (gfiumara) [inline _FIPS201/requirements.md:409]:
```suggestion
The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records of the PIV enrollment record or to the biometric data records of the PIV Card using BIO-A or OCC-AUTH authentication mechanism. Minimum accuracy requirements for
```

File: _FIPS201/requirements.md [hunk 9]
```diff
@@ -497,7 +498,7 @@ remotely via a general computing platform or a supervised remote identity proofi
 In person
 : When PIN reset is performed in person at the issuing facility, before providing the reset PIV Card
     back to the cardholder, the issuer **SHALL** perform a biometric verification to ensure that the cardholder's
-    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored either on the PIV Card or in the PIV enrollment record. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
+    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using BIO-A or OCC-AUTH authentication mechanism. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
Comment (gfiumara) [inline _FIPS201/requirements.md:501]:
```suggestion
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```

File: _FIPS201/requirements.md [hunk 10]
```diff
@@ -516,7 +517,7 @@ Supervised remote identity proofing station
     procedure. The operator **SHALL** initiate a biometric verification to ensure that
     the cardholder's biometric characteristics captured at the station elicit a
     positive biometric verification decision when compared to biometric data
-    records stored either on the PIV Card or in the PIV enrollment record. In
+    records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card and using BIO-A or OCC-AUTH authentication mechanism. In
```
Comment (gfiumara) [inline _FIPS201/requirements.md:520]:
```suggestion
    records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card and using the BIO-A or OCC-AUTH authentication mechanisms. In
```

File: _FIPS201/requirements.md [hunk 11]
```diff
@@ -406,13 +406,14 @@ If the expiration date of the new PIV Card is later than the expiration date of
...
 
-The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records obtained from either the PIV Card or PIV enrollment record. Minimum accuracy requirements for
+The issuer **SHALL** perform a biometric verification of the applicant to the biometric data records of the PIV enrollment record or to the biometric data records of the  PIV Card and performing BIO-A  or OCC-AUTH authentication mechanism. Minimum accuracy requirements for
 the biometric verification are specified in [[SP 800-76]](../_Appendix/references.md#ref-SP-800-76). On a positive biometric verification decision, the new PIV Card **SHALL** be
 released to the applicant. If the biometric verification decision is negative, or if no biometric data records are available, the cardholder
 **SHALL** provide two identity source documents (as specified in [Section 2.7](requirements.md#s-2-7)), and an attending operator **SHALL**
 inspect these and compare the cardholder with the electronic facial image retrieved from the enrollment data record
 and the photograph printed on the new PIV Card.
 
+
```
Comment (gfiumara) [inline _FIPS201/requirements.md:416]:
```suggestion
```

File: _FIPS201/requirements.md [hunk 12]
```diff
@@ -507,7 +508,7 @@ Issuer-operated kiosk
     that the cardholder's biometric characteristics elicit a positive biometric verification decision when compared to either the stored biometric on the PIV Card through an on-card one-to-one comparison or biometric data records stored in the PIV enrollment record through an off-card one-to-one
     comparison. If the biometric verification decision is negative, the cardholder's biometric characteristics are not successfully acquired, or card authentication is unsuccessful, the kiosk **SHALL NOT**
     reset the PIV Card. The session **SHALL** be terminated and the PIN reset **SHALL** be performed in person at the issuing facility or at a supervised remote identity proofing station.
-    The kiosk **MAY** be unattended while used for PIN reset operations.
+    The kiosk **MAY** be unattended while used for PIN reset operations.
```
Comment (gfiumara) [inline _FIPS201/requirements.md:511]:
```suggestion
    The kiosk **MAY** be unattended while used for PIN reset operations.
```

**Pull request merged**